Source/DafnyStandardLibraries/examples/StringsExamples.dfy

```diff
@@ -0,0 +1,68 @@
+module StringsExamples {
+  import opened DafnyStdLibs.Strings
+  import opened DafnyStdLibs.Wrappers
+
+  method {:test} TestOfInt() {
+    expect OfInt(0) == "0";
+    expect OfInt(3) == "3";
+    expect OfInt(302) == "302";
+    expect OfInt(-3) == "-3";
+    expect OfInt(-302) == "-302";
+  }
+
+  // TODO The requires clauses of these calls cause the verifier to hang, even though they seem rather trivial
+  // method {:test} TestToInt() {
```

Comment:
To be more accurate, with the configured resource limit of 1000 they run out of resources, but if I increase the resources the verification does not respond after several minutes. The problematic part of the requires clause that does not want to verify is that each symbol from the input in a key in `charMap`. Even doing `assert '0' in DecimalConversion.charMap` causes verification to hang, which to me seems like an extremely tiny steps given the definition of `DecimalConversion.charMap`: `const charMap :=
      map[
        '0' := 0, '1' := 1, '2' := 2, '3' := 3, '4' := 4, '5' := 5, '6' := 6, '7' := 7, '8' := 8, '9' := 9
      ]